unicore/ask/service/models.py

```diff
@@ -34,13 +35,22 @@ class QuestionResponse(Base, UUIDMixin):
     question_option_id = Column(
         UUIDType(binary=False), ForeignKey('question_options.uuid'),
         nullable=False)
+    created_at = Column(DateTime, default=datetime.utcnow())
```

Comment:
Won't `datetime.utcnow()` return a fixed datetime object? I think you can pass `datetime.utcnow`, but it might be better to make the database do it instead: https://github.com/universalcore/unicore.hub/blob/develop/unicore/hub/service/models.py#L47